<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateServerDialog</class>
 <widget class="QDialog" name="CreateServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="GeneralGameSetupPanel" name="generalSetupPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Rules</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="GameRulesPanel" name="rulesPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="MiscServerSetupPanel" name="miscPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFlags">
      <attribute name="title">
       <string>Flags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="DMFlagsPanel" name="dmflagsPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustomParameters">
      <attribute name="title">
       <string>Custom parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="CustomParamsPanel" name="customParamsPanel" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="toolTip">
        <string>Load previously saved server configuration.</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="toolTip">
        <string>Save current server configuration.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../../../media/resources.qrc">
         <normaloff>:/icons/diskette.png</normaloff>:/icons/diskette.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlayOffline">
       <property name="text">
        <string>Play offline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCommandLine">
       <property name="toolTip">
        <string>Obtain command line required to launch this server.</string>
       </property>
       <property name="text">
        <string>Command Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartServer">
       <property name="text">
        <string>Start server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomParamsPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/customparamspanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MiscServerSetupPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/miscserversetuppanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameRulesPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/gamerulespanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DMFlagsPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/dmflagspanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneralGameSetupPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/generalgamesetuppanel.h</header>
   <container>1</container>
   <slots>
    <signal>gameModeChanged(GameMode)</signal>
    <signal>pluginChanged(EnginePlugin*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../media/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>generalSetupPanel</sender>
   <signal>gameModeChanged(GameMode)</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>initGamemodeSpecific(GameMode)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalSetupPanel</sender>
   <signal>pluginChanged(EnginePlugin*)</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>initEngineSpecific(EnginePlugin*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initGamemodeSpecific(GameMode)</slot>
  <slot>initEngineSpecific(EnginePlugin*)</slot>
 </slots>
</ui>
